### ISSUE TITLE
Question about the training.

### ISSUE THREAD

**Issue by congwei1230** [TencentARC/BrushNet#40]
Thanks for the great work! Regarding the `segmentation_mask_brushnet_ckpt_sdxl_v0 `and `random_mask_brushnet_ckpt_sdxl_v0` provided on Google Drive, Could you please provide us with information about the number of training steps completed for each of these checkpoints? I assume that these models are trained with a batch size of 1 and a gradient accumulation of 4 on 8 GPUs. Thank you! @juxuan27
